HHS/Head-Start-TTADP
# [TTAHUB-3608] Exclude courses that have been replaced from query

## Description of change
Updated courses were still showing in the admin interface because the mapsTo construct was not accounted for anywhere. I have added that as a scope in the relevant queries.

## How to test
Perform the course upload with the CSV in the Jira ticket on prod data. You should see the correct number of courses in the admin course display.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3608


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
